run_format_tests.py

```diff
@@ -63,6 +63,7 @@ def check_format() -> None:
         'work area',
         '.eggs', '_cache',              # e.g. .mypy_cache
         'venv',                         # virtualenvs have DOS line endings
+        'test cases',
```

Comment:
This is effectively the same as removing `'.build'` from the list of suffixes to check, please don't do this
 The check exists for a reason.

Reply:
The reason I added it was because the test case has a tab character in it. I can back out that change and remove the tab from the string.

Reply:
And as a note, I still need to update the docs from the other commit to reflect the version in which they became supported.